recipe/meta.yaml

```diff
@@ -40,12 +40,12 @@ test:
     - ansi2html
   commands:
     - pip check
-    - nosetests tests/test_ansi2html.py
+    - nosetests tests/test_ansi2html.py  # [py<312]
     - ansi2html --help
     - ansi2html --version | grep -q {{ version }}
   requires:
     - mock >=0.8
-    - nose
+    - nose  # [py<312]
```

Comment:
Again, we could go with pytest and pytest-cov as specific [here](https://github.com/pycontribs/ansi2html/blob/b4224cd6a36e99117d09ad365d5a180ebf29e356/setup.cfg#L51C2-L51C2) and [here](https://github.com/pycontribs/ansi2html/blob/b4224cd6a36e99117d09ad365d5a180ebf29e356/tox.ini#L19).